shopping-cart/app/shoppingcart/ErrorBean.java

```diff
@@ -1,4 +1,4 @@
-package purchase;
```

Comment:
Yep, that's why I left them in the root package, because I plan on moving them to a common place https://github.com/sphereio/sphere-sunrise/issues/232 :)